compilesketches/compilesketches.py

```diff
@@ -909,6 +912,14 @@ class CompilationResult:
 
         if not CompilationResult.success:
             print("::error::Compilation failed")
+        else:
+            time_summary = ""
+            if diff_time > 60:
+                if diff_time > 360:
+                    time_summary += f"{int(diff_time / 360)}h "
+                time_summary += f"{int(diff_time / 60) % 60}m "
+            time_summary += f"{int(diff_time) % 60}s"
+            print("Compilation time elapsed", time_summary)
```

Comment:
```suggestion
            print("Compilation time elapsed:", time_summary)
```
Add a colon to match the format of the other compilation data output:

```text
Change in flash: 0 (0.0%)
Change in RAM for global variables: 0 (0.0%)
Change in compiler warning count: 0
```